lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -227,6 +227,8 @@ class DocumentImpl extends NodeImpl {
     }
 
     this._contentType = privateData.options.contentType;
+    this._encoding = privateData.options.encoding;
```

Comment:
The default encoding should depend on language (https://html.spec.whatwg.org/#determining-the-character-encoding:concept-encoding-confidence-8) but I didn't want to add it now. I can remove the parameter but I will have to change some tests.